stylesheets/blue/components/_button_primary.scss

```diff
@@ -4,6 +4,7 @@
 //
 // default - Normal state
 // .ButtonPrimary--fullWidth - A button that takes the whole width of its parent
+// .ButtonPrimary--brightTurquoise - A button that uses the brightTurquoise color
```

Comment:
Well you you know I disagree.
A generic/functional name resists less to future changes.
A specific/visual-inspired name resists a lot more.

So why don't you find this an advantage?